cumulusci/robotframework/Performance.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
```

Comment:
Filename: `performance.py`. (Unless there is a Robot reason we need to use non-snake-case).

Reply:
When importing libraries, robot wants the filename to be exactly the same as the classname. For BaseLibrary it doesn't matter since we don't import that through a robot file, but for something directly imported from a robot test, this is the standard convention. It's workaround-able, but I kept it camel case to be consistent with the other robot libraries (Salesforce.py, Salesforce.robot).

Reply:
Understood. Comment withdrawn.